web/src/topic_generator.js

```diff
@@ -81,7 +81,6 @@ export function get_next_topic(curr_stream, curr_topic, only_followed_topics) {
         const topics = stream_topic_history.get_recent_topic_names(stream_id);
 
         if (
-            narrow_state.active() &&
             narrow_state.stream_id() === stream_id &&
```

Comment:
I'm not sure this change is correct; `stream_data.get_stream_id` can return `undefined` for an invalid stream name, so I think this may cause this code to run when `narrow_state.stream_id` is undefined with an invalid stream name in some cases?

Reply:
Changed the condition in #28812